server/src/api/api.js

```diff
@@ -1,3 +1,72 @@
 'use strict';
 
 const router = require('../lib/router.js');
+const Notes = require('../models/notes');
+
+let sendJSON = (res, data) => {
+  res.statusCode = '200';
+  res.statusMessage = 'OK';
+  res.setHeader('Content-type', 'application/json');
+  res.write(JSON.stringify(data));
+  res.end();
+};
+
+let serverError = (res, err) => {
+  let error = { error: err };
+  res.statusCode = 500;
+  res.statusMessage = 'Server Error';
+  res.setHeader('Content-Type', 'application/json');
+  res.wrote(JSON.stringify(error));
```

Comment:
Don't know if you'll catch this or not but you'll probably want to fix the typo here "wrote" should be "write"